todo-list/src/Components/AddTask/AddTask.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+
+import "./AddTask.css";
+import Plus from "../../Images/plus.png";
+
+function AddTask({ onAddTodo }) {
+  const [isClicked, setIsClicked] = useState(false);
+  function clickHandler(e) {
+    !isClicked ? setIsClicked(true) : setIsClicked(false);
+  }
+  const [enteredTodo, setEnteredTodo] = useState([]);
+  //  form submission
+  function submitHandler(event) {
+    const { preventDefault } = event;
+    preventDefault();
+    // error handling for not giving input
+    if (enteredTodo === "") {
+      alert("Please enter the todo.");
+    } else {
+      const defaultTodos = {
+        id: Math.random().toString(),
+        title: enteredTodo,
+      };
+      onAddTodo(defaultTodos);
+      setEnteredTodo("");
+    }
+  }
+  return (
+    <div className="add-task-container">
```

Comment:
In Practical-4 you just show the default task no need to add add/create new task functionality